### ISSUE TITLE
Switch Nginx config to use $http_host

### ISSUE THREAD

**Issue by ActionScripted** [realpython/dockerizing-django#28]
Great, short explanation:
http://stackoverflow.com/a/20752916/12276

I was running into errors with a slightly more complex Django setup of my own and this fixed things.

**Comment by mjhea0** [comment]
Are you suggesting I change the nginx config?

**Comment by ActionScripted** [comment]
Yes, sorry for not being more clear or submitting a proper PR. Switching line 13 in nginx/sites-enabled/django_project from `proxy_set_header Host $host;` to `proxy_set_header Host $http_host;` will keep folks using frontnend forms and such from running into errors.

**Comment by mjhea0** [comment]
Can you submit a quick PR? :)

**Comment by ActionScripted** [comment]
Wanted to be sure I wasn't missing something and in double-checking things I found that in `docker-compose.yml` I was mapping Nginx's port as "8080:80" which was causing problems.

`$http_host` fixed this but that was only treating a symptom. After reading more about whether `$http_host` was preferred to `$host` I've learned that `$host` is absolutely the way to go 99% of the time.

Closing this because it was a problem with my config and nothing with this repo. Thanks so much for being so polite and for the great article.

Related information:
- https://forum.nginx.org/read.php?2,213799,213813#msg-213813
- http://stackoverflow.com/a/15414811/12276
- http://serverfault.com/a/393040/203743
